frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/object-bucket-claim.tsx

```diff
@@ -67,7 +68,14 @@ const OBCTableRow: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj }) => {
         {storageClassName ? <ResourceLink kind="StorageClass" name={storageClassName} /> : '-'}
       </TableData>
       <TableData className={tableColumnClasses[5]}>
-        <ResourceKebab actions={menuActions} kind={kind} resource={obj} />
+        <ResourceKebab
```

Comment:
is this not yet migrated to ODF-Console?

Reply:
Nope. Kebab implementation is a bit different there.

Reply:
ack